cli/azd/test/functional/cli_test.go

```diff
@@ -694,8 +694,6 @@ func newTestContext(t *testing.T) (context.Context, context.CancelFunc) {
 }
 
 func Test_CLI_InfraCreateAndDeleteResourceTerraform(t *testing.T) {
-	// running this test in parallel is ok as it uses a t.TempDir()
```

Comment:
Did you intend to include this?

Reply:
Yes, terraform tests are inconsistent if running in parallel. @wbreza is looking into it.
But this should help make CI a little more stable